python/tester/upload_from_repo.py

```diff
@@ -189,7 +201,7 @@ def prepare_one_visit(kafka_url, group_id, butler, instrument, visit_id):
             rotationSystem=SummitVisit.RotSys.SKY,
             cameraAngle=data_id.records["exposure"].sky_angle,
             survey="SURVEY",
-            salIndex=999,
+            salIndex=INSTRUMENTS[instrument].sal_index,
             scriptSalIndex=999,
```

Comment:
So far we don't use `scriptSalIndex` for anything and nothing in prompt processing cares what value it has.   So it can even be a random number and it doesn't matter.